src/pages/components/table.mdx

```diff
@@ -193,6 +193,51 @@ import TableVirtualizedCode from '!!raw-loader!../../examples/table/TableVirtual
 
 ## API
 
+The Table component follows this structure:
+
+```jsx
+<Table>
+  <Caption />
+  <Head>
+    <HeaderRow>
+      <HeaderCell />
+      <HeaderCell />
+      <SortableCell /> {/* Optional */}
+      {/* etc. */}
+    </HeaderRow>
+  </Head>
+  <Body>
+    <GroupRow>
+      {' '}
+      {/* Optional */}
+      <Cell />
+    </GroupRow>
+    <Row>
+      <Cell />
+      <Cell />
+      {/* etc. */}
+    </Row>
+    {/* etc. */}
+  </Body>
+</Table>
+```
+
+<!-- markdownlint-disable no-emphasis-as-heading -->
+
+**Implementation notes**
+
+<!-- markdownlint-enable no-emphasis-as-heading -->
+
+<!-- markdownlint-disable line-length -->
+
+- Follow the
+  [MDN Table Accessibility Practices](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#Tables_for_visually_impaired_users)
+  guidelines to ensure the table data is accessible to screen-readers
+- Use the [SortableCell](#sortablecell) component in a sortable table to ensure that the header is interactive
+  and accessible. See an [example](#sort) usage.
+
+<!-- markdownlint-enable line-length -->
+
 ### Body
 
```

Comment:
Out of curiosity: where would folks need to go to learn the native table elements structure? I'm curious how Body and Cell are described. (We were having naming issues for James' patterns documentation on what to call cells that aren't header cells for context).

Since this part of the documentation is primarily for devs, I trust your instinct if it feels repetitive.